tbx/work/models.py

```diff
@@ -227,17 +234,17 @@ def first_author(self):
     def related_works(self):
         return [
             {
-                "client": case_study.client,
-                "title": case_study.title,
-                "url": case_study.url,
-                "author": case_study.first_author,
-                "date": case_study.date,
-                "read_time": case_study.read_time,
-                "related_services": case_study.related_services.all(),
-                "listing_image": case_study.header_image,
+                "client": work_page.client,
```

Comment:
@engineervix I renamed these to keep the naming consistent

Reply:
Nice one, thank you!